apps/fluent-tester/src/TestComponents/ActivityIndicator/ActivityIndicatorTest.tsx

```diff
@@ -5,24 +5,33 @@ import { Stack } from '@fluentui-react-native/stack';
 import { stackStyle, commonTestStyles as commonStyles } from '../Common/styles';
 import { Test, TestSection, PlatformStatus } from '../Test';
 import { ACTIVITY_INDICATOR_TESTPAGE } from './consts';
-import { View, Switch } from 'react-native';
+import { View } from 'react-native';
+import { Switch } from '@fluentui-react-native/switch';
+import { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
 
 const BasicActivityIndicator: React.FunctionComponent = () => {
   const [animating, setAnimating] = React.useState(true);
   const [hidesWhenStopped, setHidesWhenStopped] = React.useState(true);
 
+  const onChangeAnimating = React.useCallback(
+    (__e: InteractionEvent, value) => {
+      setAnimating(value);
+    },
+    [setAnimating],
+  );
```

Comment:
I think these methods for Switch state change can be combined to one and `setSmth` can be passed as a parameter